app/models/hackbot/conversations/check_in.rb

```diff
@@ -10,13 +10,24 @@ def self.should_start?(event, _team)
         event[:text] == 'check in'
       end
 
+      # rubocop:disable Metrics/LineLength, Metrics/MethodLength
       def start(event)
-        first_name = leader(event).name.split(' ').first
-
-        msg_channel "Hey #{first_name}! Did you have a club meeting this week?"
+        leader_info = leader(event)
+        first_name = leader_info.name.split(' ').first
+
+        if first_check_in?
+          msg_channel "Hi #{first_name}! I'm Hackbot - Hack Club's friendly robotic helper. "\
+            "Right now I'm reaching out to you so I can collect some data about your club's "\
+            'activity in the past week so that the team can improve your experience working '\
+            'with them :) '\
+            'So, first question: Did you have a club meeting this week?'
```

Comment:
Couple things here.

- Do you want this to be one giant block of text? Or broken up with new lines? Currently a giant block of text.
- Let's use emoji instead of text smiley faces with Hackbot.

Copy can also be improved, that sentence uses the word "so" a lot right now.

> Hey Harrison! I'm Hackbot, Hack Club's friendly robot helper.
>
> I'll be reaching out to you every week, typically on Fridays, to check in and see how your club's doing. I'll be sharing everything with the team, so they'll be in the loop every step of the way 🙂.
>
> To start, did you have a club meeting this week?